src/PlusDataCollection/USDigitalEncodersTracking/vtkPlusUSDigitalEncodersTracker.h

```diff
@@ -146,11 +146,11 @@ class vtkPlusDataCollectionExport vtkPlusUSDigitalEncodersTracker : public vtkPl
 
 
 private:  // Variables.
-    bool coreXY;
+    bool coreXY = false;
     bool configurationViaSerialNumbers = false;
-    long NumberOfEncoders; // Number of connected A2 Encoders
-    long COMPort;
-    long BaudRate;
+    long NumberOfEncoders = 0; // Number of connected A2 Encoders
+    long COMPort = 0;
+    long BaudRate = 9600;

```

Comment:
Also, I know we can recite these in our sleep, but would you consider seeing if there's an enum to re-use (can move to PlusCommon if there's another device/class using the same values)?